content/quickstart/azure/configure.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Configure Azure
+weight: 4
+menu:
+  quickstart:
+    parent: azure
+    identifier: azure-configure
+---
+
+<!-- TODO inline a streamlined version of configuring the cloud here. -->
+
+<a href="{{< relref "/reference/clouds/azure/setup.md" >}}" target="_blank">Configure Azure</a> so the Pulumi CLI can connect to Azure.
```

Comment:
Similar to my command above, I would say that if the Azure CLI, `az`, is configured, Pulumi will use its settings.